gui/options.cpp

```diff
@@ -1619,7 +1619,7 @@ void GlobalOptionsDialog::build() {
 
 	// Select the currently configured language or default/English if
 	// nothing is specified.
-	if (ConfMan.hasKey("gui_language"))
+	if (ConfMan.hasKey("gui_language") && ConfMan.get("gui_language") != "")
```

Comment:
That was indeed one of three easy way to fix the issue. I think this is the one I like the least, but it is also the less intrusive and safer. So this is fine with me.
However, rather than test equality with an empty string, you should use `bool Common::String::empty()`.